library-docker/bin/start.sh

```diff
@@ -1,56 +1,49 @@
 #!/bin/bash
 
-SERVER_PID=
-SCRIPT_PID=$$
-echo "Script PID is $SCRIPT_PID"
-
-start () {
-  echo "Starting kiwix-serve..."
-  kiwix-serve --attachToProcess=$SCRIPT_PID --port=80 --library --threads=16 --verbose --nodatealias library.kiwix.org.xml
-  SERVER_PID=$!
-  return $SERVER_PID
-}
-
-stop () {
-  echo "Stopping kiwix-serve..."
-  kill -9 $SERVER_PID
-  return
-}
-
-is_alive () {
-  echo "Testing kiwix-serve..."
-  timeout 50 curl http://localhost/catalog/searchdescription.xml > /dev/null 2>&1
-  return $?
-}
-
-{ \
-  #  echo "User-agent: *" ; \
-  #  echo "Crawl-delay: 3" ; \
-  echo "User-agent: *" ; \
-  echo "Disallow: /" ; \
-} > /var/www/library.kiwix.org/robots.txt
-
-{ \
-  echo "#!/bin/sh" ; \
-  echo "cd $LIBRARY_DIR" ; \
-  echo "manageLibraryKiwixOrg.pl --source=/var/www/download.kiwix.org/library/library_zim.xml >library.kiwix.org.xml 2>>/dev/shm/libgen" ; \
-  echo "kill \`pidof kiwix-serve\`" ; \
-} > /etc/cron.daily/80generateLibraryKiwixOrg && chmod 0500 /etc/cron.daily/80generateLibraryKiwixOrg
-
-{ \
-  echo "#!/bin/sh" ; \
-  echo "cd $LIBRARY_DIR" ; \
-  echo "manageContentRepository.pl --writeWiki --writeRedirects --writeHtaccess --writeLibrary --deleteOutdatedFiles >>/dev/shm/libgen 2>&1" ; \
-  echo "updateWikiPage.py wiki.kiwix.org LibraryBot `cat /run/secrets/wiki-password` 'Template:ZIMdumps/content' '/var/www/download.kiwix.org/zim/.contentPage.wiki' 'Automatic update of the ZIM library'" ; \
-} > /etc/cron.daily/10updateContentRepository && chmod 0500 /etc/cron.daily/10updateContentRepository
+printf "
+# User-agent: *
+# Crawl-delay: 3
+User-agent: *
```

Comment:
We could probably remove that line (my error).

Reply:
Yeah I figured you wanted to keep a reference to it in case the denial was temporary. robots.txt supports comment but will removed them both.

Reply:
I was talking just about `User-agent: *`